pipeline-maven-api/src/main/java/org/jenkinsci/plugins/pipeline/maven/dao/PipelineMavenPluginNullDao.java

```diff
@@ -40,12 +43,34 @@
 /**
  * @author <a href="mailto:cleclerc@cloudbees.com">Cyrille Le Clerc</a>
  */
+@Extension
 public class PipelineMavenPluginNullDao implements PipelineMavenPluginDao {
     private static Logger LOGGER = Logger.getLogger(PipelineMavenPluginNullDao.class.getName());
 
+    @Override
+    public String getDescription() {
+        // TODO i18n
```

Comment:
could you please fix those i18n TODOs ?

Reply:
There are so many places i18n should be fixed, and there is not even a single TODO since the project has been started because this has not even been taken into account.
At least there is one TODO here. If I didn't put the TODO not sure someone will have noticed....
Frankly fix one place but not all the others not sure this is worth it.

Reply:
anyway I did it...

Reply:
you are right, as you said, there is so many things to do in this plugin, at so many places ...
I would just like to avoid increase the work to do while working on improvements or new features

Many thanks for doing it !